src/CMakeLists.txt

```diff
@@ -295,6 +295,10 @@ if(APPLE)
   target_sources(shotcut PRIVATE ${APP_ICON})
   set_source_files_properties(${APP_ICON} PROPERTIES MACOSX_PACKAGE_LOCATION "Resources")
 
+  #Create a symlink to the qml folder after building
+  file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/Shotcut.app/Contents/Resources/qml)
```

Comment:
MAKE_DIRECTORY should not have "/qml" at the end as that is the symlink in the next line.

Reply:
Yes, my bad. Will fix it.